README.md

```diff
@@ -349,6 +349,16 @@ all of your arguments must be keyword arguments.
 Note: This is a limitation of the current `significant_kwargs` implementation,
 and totally fixable if someone wants to submit a pull request.
 
+### async_or_eager
```

Comment:
I think listing the two `async_` versions after the `delay_` versions still makes sense.